tests/functional/disaster-recovery/regional-dr/test_acm_kubevirt_dr_integration.py

```diff
@@ -0,0 +1,331 @@
+            cnv_workloads[0].workload_namespace,
+            discovered_apps=True,
+            resource_name=resource_name,
+        )
+
+        verify_drpolicy_ui(acm_obj, scheduling_interval=scheduling_interval)
+
+        primary_cluster_name_before_failover = (
+            dr_helpers.get_current_primary_cluster_name(
+                cnv_workloads[0].workload_namespace,
+                discovered_apps=True,
+                resource_name=resource_name,
+            )
+        )
+        logger.info(
+            f"Primary cluster name before failover is {primary_cluster_name_before_failover}"
+        )
+
+        config.switch_to_cluster_by_name(primary_cluster_name_before_failover)
+        dr_helpers.wait_for_all_resources_creation(
+            cnv_workloads[0].workload_pvc_count * 2,
+            cnv_workloads[0].workload_pod_count * 2,
+            cnv_workloads[0].workload_namespace,
+            discovered_apps=True,
+            vrg_name=resource_name,
+        )
+        dr_helpers.wait_for_cnv_workload(
+            vm_name=cnv_workloads[0].vm_name,
+            namespace=cnv_workloads[0].workload_namespace,
+            phase=constants.STATUS_RUNNING,
+        )
+
+        config.switch_to_cluster_by_name(primary_cluster_name_before_failover)
+        # Download and extract the virtctl binary to bin_dir. Skips if already present.
+        CNVInstaller().download_and_extract_virtctl_binary()
+        secondary_cluster_name = dr_helpers.get_current_secondary_cluster_name(
+            cnv_workloads[0].workload_namespace,
+            discovered_apps=True,
+            resource_name=resource_name,
+        )
+
+        # Creating a file (file1) on VM and calculating its MD5sum
+        for cnv_wl in cnv_workloads:
+            md5sum_original.append(
+                run_dd_io(
+                    vm_obj=cnv_wl.vm_obj,
+                    file_path=vm_filepaths[0],
+                    username=cnv_wl.vm_username,
+                    verify=True,
+                )
+            )
+
+        for cnv_wl, md5sum in zip(cnv_workloads, md5sum_original):
+            logger.info(
+                f"Original checksum of file {vm_filepaths[0]} on VM {cnv_wl.workload_name}: {md5sum}"
+            )
+
+        wait_time = 2 * scheduling_interval  # Time in minutes
+        logger.info(f"Waiting for {wait_time} minutes to run IOs")
+        sleep(wait_time * 60)
+
+        # Shutdown primary managed cluster nodes
+        active_primary_index = config.cur_index
+        active_primary_cluster_node_objs = get_node_objs()
+        logger.info("Shutting down all the nodes of the primary managed cluster")
+        nodes_multicluster[active_primary_index].stop_nodes(
+            active_primary_cluster_node_objs
+        )
+        logger.info(
+            f"All nodes of the primary managed cluster {primary_cluster_name_before_failover} are powered off, "
+            "waiting for cluster to be unreachable.."
+        )
+        sleep(300)
```

Comment:
should we wait for nodes to be unreachable instead of sleeping for 5 minutes. i.e., write an inverse method of `wait_for_nodes_status`?

Reply:
is the cluster being marked as `unavailable` is only possible to check via UI?